---------------------
PatchSet 319 
Date: 2010/09/25 06:13:55
Author: millert
Branch: HEAD
Tag: (none) 
Log:
Add CVE-2010-0405 to list of local patches.

Members: 
	git_version.h:1.2->1.3 
	patchlevel.h:1.18->1.19 

Index: src/gnu/usr.bin/perl/git_version.h
diff -u src/gnu/usr.bin/perl/git_version.h:1.2 src/gnu/usr.bin/perl/git_version.h:1.3
--- git_version.h:1.2	Fri Sep 24 15:06:40 2010
+++ git_version.h	Sat Sep 25 13:13:55 2010
@@ -2,7 +2,6 @@
 * WARNING: 'git_version.h' is automatically generated by make_patchnum.pl
 *          DO NOT EDIT DIRECTLY - edit make_patchnum.pl instead
 ***************************************************************************/
-#define PERL_GIT_UNCOMMITTED_CHANGES 
 #define PERL_PATCHNUM ""
 #define PERL_GIT_UNPUSHED_COMMITS		\
     /*leave-this-comment*/
Index: src/gnu/usr.bin/perl/patchlevel.h
diff -u src/gnu/usr.bin/perl/patchlevel.h:1.18 src/gnu/usr.bin/perl/patchlevel.h:1.19
--- patchlevel.h:1.18	Fri Sep 24 15:06:40 2010
+++ patchlevel.h	Sat Sep 25 13:13:55 2010
@@ -127,6 +127,7 @@
 #  endif
 static const char * const local_patches[] = {
 	NULL
+	,"CVE-2010-0405"
 #ifdef PERL_GIT_UNCOMMITTED_CHANGES
 	,"uncommitted-changes"
 #endif
